config/flat.py

```diff
@@ -1,5 +1,5 @@
-import os.path
 from lsst.utils import getPackageDir
+import os.path
```

Comment:
System imports go before LSST/PFS ones.  Please check all changesets and fix (and squash)